packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -25,11 +25,34 @@
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+import { isBoolean, isString } from 'lodash'
+
+import { isYocoIcon } from '../validators'
+
+import { KottiBreadcrumb } from './types'
+
+const breadcrumbValidator = (
+	value: KottiBreadcrumb.Breadcrumb,
+): value is KottiBreadcrumb.Breadcrumb =>
+	value !== null &&
+	typeof value === 'object' &&
+	isBoolean(value.isCompleted) &&
+	isString(value.title) &&
+	isString(value.to)
+
+export default defineComponent<KottiBreadcrumb.PropsInternal>({
 	name: 'KtBreadcrumb',
 	props: {
-		breadcrumbs: { type: Array, required: true },
+		breadcrumbs: {
+			type: Array,
+			required: true,
+			validator: (value: unknown) =>
+				Array.isArray(value) &&
+				value.length > 0 &&
+				value.every(breadcrumbValidator),
+		},
 		separator: {
 			type: Object,
 			default() {
```

Comment:
I can't comment on https://github.com/3YOURMIND/kotti/pull/415/files#diff-2b4d6c2a4746f4f5fdeda076d4b86e007548a025cb9d9c1dc1bf8e50dcb6c7b3R60 but I'd make an `enum` of the style 

and expose this on the types as well ?
or at least expose the type for `Kotti.BreadCrumb.Seperator`to be `{ style: 'icon', value: Yoco.Icon } | { style: 'text', value: 'string' }`

Reply:
Already done, nvm 👏🏼